packages/common/src/LocaleUtils.ts

```diff
@@ -35,7 +35,7 @@ export const getBloodTypeTranslation = (bloodType: BloodType) => {
     case BloodType.O_PLUS:
       return "O+";
     case BloodType.NOT_SURE:
-      return "לא יודע";
+      return "לא יודע/ת";
```

Comment:
Didn't you just change this line in a different PR?

Reply:
Will be solved in merge